packages/neos-ui-ckeditor5-bindings/tests/manual/index.html

```diff
@@ -0,0 +1,21 @@
+<html lang="en">
```

Comment:
i added some manual tests, which can be used to test our ckeditor integration without a fully fledged neos